examples/long-short.js

```diff
@@ -0,0 +1,423 @@
+const API_KEY = 'PK2QQWOEG6ALQ0ADZV3H';
+const API_SECRET = 'f393ILtkPrxxnFgESMzSeDnDkFiDB9zXUZcJLjm0';
```

Comment:
Is this the real key? Regardless, it should fail clearly unless the reader sets the right key.